core/services/network_config.rb

```diff
@@ -1,16 +1,18 @@
 # frozen_string_literal: true
 
 require_relative '../node'
-require_relative 'vagrant_commands'
+require_relative 'machine_commands'
 require 'stringio'
 
-# Network configurator for vagrant nodes
+# Network configurator for vagrant and terraform nodes
 class NetworkConfig
```

Comment:
You must not use the NetworkConfig class during the implementation of the Terraform for AWS.